test/utils/bytecode.hpp

```diff
@@ -242,6 +244,24 @@ inline bytecode jumpi(bytecode target, bytecode condition)
     return condition + target + OP_JUMPI;
 }
 
+inline bytecode rjump(int16_t offset)
+{
+    return OP_RJUMP + big_endian(offset);
+}
+
+inline bytecode rjumpi(int16_t offset, bytecode condition)
+{
+    return condition + OP_RJUMPI + bytecode{big_endian(offset)};
+}
+
+inline bytecode rjumpv(const std::initializer_list<uint16_t> offsets, bytecode condition)
```

Comment:
sorry this was wrong
```suggestion
inline bytecode rjumpv(const std::initializer_list<int16_t> offsets, bytecode condition)
```